controllers/manila_controller.go

```diff
@@ -655,7 +656,24 @@ func (r *ManilaReconciler) reconcileNormal(ctx context.Context, instance *manila
 			err.Error()))
 		return ctrl.Result{}, err
 	}
-	if op != controllerutil.OperationResultNone {
+	apiObsGen, err := r.checkManilaAPIGeneration(instance)
+	if err != nil {
+		instance.Status.Conditions.Set(condition.FalseCondition(
+			manilav1beta1.ManilaAPIReadyCondition,
+			condition.ErrorReason,
+			condition.SeverityWarning,
+			manilav1beta1.ManilaAPIReadyErrorMessage,
+			err.Error()))
+		return ctrlResult, nil
+	}
+	if !apiObsGen {
```

Comment:
we need to make the below ready count update and mirroring conditional.

same for the other subCRs below as well

Reply:
yes, updating this patch and make sure is aligned with what proposed for designate and octavia